styleguide/global/global.js

```diff
@@ -32,6 +32,28 @@ jQuery(document).ready(($) => {
 })(jQuery); // to protect and scope the JQuery alias = $
 
 
+// This plugin can be used to anchor scroll to anything. Usage:
+// 1) Give your link the "rsAnchor-scroll" class
+// 2) Give the href of your link a hashed id (ie href="#someID")
+// 3) Set that id to the element you want scrolled to (ie id="someID")
+(($) => {
+  $.fn.anchorScroll = function anchorScroll() {
+    $(this).click(function scrollTo(e) {
+      e.preventDefault();
+      // we need to subtract the top nav or else scrolled section will look cut off.
+      const fixedNav = $('.navbar-fixed-top').outerHeight();
+      const $this = $(this.hash);
+      if (typeof $this.offset() !== typeof undefined && $this.offset() !== false) {
+        $('html, body').animate({
+          scrollTop: ($this.offset().top - fixedNav),
+        }, 500);
+      }
+    });
+    return this;
+  };
+})(jQuery);
```

Comment:
I put this one in global because it's not page specific. It's a plugin we can use globally to anchor scroll to anything, not just in customer stories.